### PR TITLE
Tanzu java web app gradle

### PR DESCRIPTION
**Pull request by kdvolder**
Adding gradle support to tanzu-java-web-app

### CODE REVIEW

File: create-acc.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (trisberg) [inline create-acc.sh:1]: I don't think we need this file in the repo

File: run-test-local.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+export GITHUB_WORKSPACE=$(pwd)
+
+docker pull dev.registry.tanzu.vmware.com/app-accelerator/acc-engine:latest
```
Comment (trisberg) [inline run-test-local.sh:6]: this is not publicly accessible, we should not commit this to the repo

**Review by trisberg — CHANGES_REQUESTED** [review]
I created a separate PR #256 for the script to run local tests. The other `create-acc.sh` file can be removed.

File: run-test-local.sh (hunk 2 above)
Comment (kdvolder) [reply run-test-local.sh]: Merged your changes from other PR and put the docker commands in separate script. I hope that's okay. If you are objecting to the mere presence of the string`dev.registry.tanzu.vmware.com/app-accelerator/acc-engine:latest` anywhere in the repo... I will point out that this string can also be found inside the .github workflow definitions.

**Review by trisberg — APPROVED** [review]

**Comment by trisberg** [comment]
squashed and merged as f79fc00f08e505f6966032b4f4aba54fafbd054c

**Pull request closed without merging**